microcosm_eventsource/stores/rollup.py

```diff
@@ -88,6 +88,13 @@ def search(self, **kwargs):
             for row in self._search_query(aggregate, **kwargs).all()
         ]
 
+    def search_first(self, **kwargs):
```

Comment:
maybe we can use the existing `search` function with `limit=1`?

Reply:
that would return an array

Reply:
and add `res[0]`

Reply:
which will throw an exception instead of returning None